spree_social.gemspec

```
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.requirements << 'none'
 
-  s.add_runtime_dependency 'solidus_core', [">= 1.1.0.pre", "<= 1.4.0"]
+  s.add_runtime_dependency 'solidus_core', [">= 1.1.0.pre", "<= 2.0"]
```

```suggestion
  s.add_runtime_dependency 'solidus_core', [">= 1.1.0.pre", "< 2.0"]
```

Did you mean strictly `< 2.0` so we could go to v1.4.2 now, and revisit before 2.0?

If you want to allow v2.0, we should go up to the latest tiny version, `2.0.3`.

---

I believe this work is to get this 💎 working with the 1.4 mainline ? Thereby doing incremental bumps to get us there. We also brought the gem version of Solidus in the Gemfile to 1.4.2 so we could be assured our test suite was passing with a 1.4 version of the app.

This causes micro updates which I agree are a little tiring but, due to how large our App is and how different it is from 1.4, being conservative here made sense. Maybe being hyperconservative wasn’t the best move?